### DISCUSSION TITLE
How to set a longer brightness transition time

### DISCUSSION THREAD

**Discussion by cndian** [Koenkk/zigbee2mqtt#11456] [Devices]
Hello,

I would like to adjust the brightness transition time of a device. How can I achieve this? I'm using Zigbee2mqtt with Domoticz and SSH to program.

The device is https://www.zigbee2mqtt.io/devices/GL-B-007P.html

Thanks in advance

**Comment by lux73** [comment]
take a look in your posted Link

`transition` - under Options...

**Comment by cndian** [comment]
Thank you, but I don't know where to change it in SSH or Domoticz.

**Reply by lux73** [reply]
in `configuration.yaml`

![z2m_transition](https://user-images.githubusercontent.com/3129990/154841814-903ab5b3-9b69-48bc-bf8d-e910a4cbab97.PNG)

be aware there must be two (2!) spaces before transition - the .yaml is very case sensitive...   if there is only one space or no space z2m will probably refuse to start...

**Comment by cndian** [comment]
Thanks it's working! Is there also an option to only set different transition time by powering off?
